### DISCUSSION TITLE
Using scopes like "roles" , " groups" from OIDC provider to bind perses roles to "OIDC groups"

### DISCUSSION THREAD

**Discussion by khimick** [perses/perses#2263] [Q&A] [answered]
Hello. Is it possible to get scopes, something like "roles, groups", from oidc provider and use group/role name to bind it to perses role?
Something like: users from group admin bind to perses group "Admin"...

**Comment by celian-garcia** [comment, marked as answer]
Hello, not yet. But this is something that we'll need at Amadeus at some point as we're under Azure and we would like to benefit it's access recertification.
